fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -330,16 +335,27 @@ private IndexingByRecords getIndexerByRecordsOrThrow() {
         return getIndexerByRecords();
     }
 
+    @Nonnull
+    private IndexingScrubMissing getIndexerScrubber() {
```

Comment:
Does this need to be removed or updated to also possibly return a `IndexingScrubDangling`?

Reply:
I've missed it during the cleanup. 
Removed.